code/modules/projectiles/projectile.dm

```diff
@@ -67,10 +67,26 @@
 	qdel(src)
 
 /obj/item/projectile/proc/on_hit(atom/target, blocked = 0, hit_zone)
+	var/turf/target_loca = get_turf(target)
 	if(!isliving(target))
 		return 0
 	var/mob/living/L = target
 	if(blocked < 100) // not completely blocked
+		if(damage && L.blood_volume && damage_type == BRUTE)

```

Comment:
no... the `damage` check alone is basically going (in pseuedocode): `if(damage > 0) return TRUE`.. so that line there is just saying 'has it done any damage?'

If you wanted it to only trigger on damage higher than 5 instead, you'd just do:

`if(damage > 5 && L.blood_volume && damage_type == BRUTE)`